en/companion_computer/integrated_companion.md

```diff
@@ -0,0 +1,22 @@
+# Integrated Flight Controller/Companion Computer
+
+Controller boards that come with a _pre-integrated_ [companion/mission computer](../companion_computer/README.md) and [flight controller](../flight_controller/README.md) are recommended for supporting computationally expensive features, such as [object avoidance](../computer_vision/obstacle_avoidance.md) and [collision prevention](../computer_vision/collision_prevention.md).
+
+If you need a companion computer, then using a pre-integrated board is likely to significantly reduce your costs.
+
+## Supported Boards
+
+The following boards are known to provide a good integration with PX4:
+
+- [Holybro Pixhawk RPI CM4 Baseboard](../companion_computer/holybro_pixhawk_rpi_cm4_baseboard.md) - Baseboard with plugin Raspberry Pi and [Pixhawk Autopilot Bus](../flight_controller/pixhawk_autopilot_bus.md) compatible flight controllers.
+
+## Software Setup
+
+The companion computer and flight controller must be set up to communicate with each other (typically using MAVLink and ROS2, over Ethernet or a serial port), and may also be set up to communicate with ground controllers, cloud and other systems.
+
+The setup depends on the companion computer and its operating system, and how the companion and flight controller are connected.
```

Comment:
Certainly it would be good if the PX4 project had a recommended companion OS and update method for that OS. That would make it possible to have turnkey instructions and CI testing that would work for 80% of the world.

Everyone seems to want their own OS though, so perhaps all that would need to be containerized?

Anyway, not my problem to solve.